Maple2.Server.Game/Manager/ExperienceManager.cs

```diff
@@ -143,6 +147,19 @@ public sealed class ExperienceManager {
         AddExp((long) ((expValue * modifier) * entry.Factor) + additionalExp, expType.Message());
     }
 
+    public void AddMobExp(int moblevel, float modifier = 1f, long additionalExp = 0) {
+        if (!session.TableMetadata.ExpTable.ExpBase.TryGetValue(2, out IReadOnlyDictionary<int, long>? expBase)) {
+            return;
+        }
+
+        if (!expBase.TryGetValue(moblevel, out long expValue)) {
+            return;
+        }
+
+
+        AddExp((long) (expValue * modifier) + additionalExp, ExpType.monster.Message());
+    }
```

Comment:
I wonder if we just combined this with the method above? I believe there is a `monster` ExpType value

And change the parameters to include `level`?

Reply:
but we dont have factor for monster

Reply:
ah yeah. nevermind